lib/Classes/common/Request.ts

```diff
@@ -133,6 +134,17 @@ class Request {
     return requestHeaders;
   }
 
+  setSubaccountHeader(subaccountId: string): void {
+    const headers = this.makeHeadersFromObject({
+      [SubaccountsClient.SUBACCOUNT_HEADER]: subaccountId
+    });
+    this.headers.set(headers);
```

Comment:
Here we may lose the original headers that were set by the user. 
Maybe better to merge headers here or as an option we can store them in a private variable until the user disables the usage of the sub-account.